src/parser/rogue/subtlety/modules/spell/shadowlands/covenants/venthyr/Flagellation.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
```

Comment:
is this a duplicate of rogue/shared?

Reply:
Yes. I committed but didn't push my deletion of these files